packages/spor-bottom-drawer-react-native/src/BottomDrawer.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { createContext, useContext, useState } from "react";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import { DrawerHandle } from "./DrawerHandle";
+import Modal from "react-native-modal";
+import { DrawerProvider } from "./DrawerContext";
+
+type BottomDrawerProps = {
+  children: React.ReactNode;
```

Comment:
Hvis denne skal være ReactNode og ikke JSX.Element så må det lages en rendring for de tilfellene hvor den er en string, og en hvor den ikke er det. Sendes det inn en string sånn som det er nå så vil ikke typescript klage, men appen vil kræsje.